src/bundle/test-transformer.ts

```diff
@@ -52,3 +65,19 @@ export const setupTransformHook = (opts: { removeNonJsImports?: boolean } = {}):
 
     return revertTransformHook;
 };
+
+function shouldIgnoreImportFromStyleFile(err: Error): boolean {
+    const shouldIgnoreImport = IGNORE_STYLE_ERRORS.some(ignoreImportErr => {
+        return (err as Error).message.startsWith(ignoreImportErr);
+    });
+
+    if (!shouldIgnoreImport) {
+        return false;
+    }
+
+    const firstStackFrame = (err as Error).stack?.split("\n")[0] || "";
+    const filePath = firstStackFrame.split(":")[0];
+    const isStyleFilePath = STYLE_EXTESTION_RE.test(filePath);
```

Comment:
When I get an error from import css first frame in stack looks like: `/Users/.../some.css:100500`.